lib/ecb_rates/application.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+
+module EcbRates
+  class Application
```

Comment:
Please, could you describe what it simplify? In my opinion usage looks slightly complicated. Like:

```ruby
raise_error EcbRates::Application::CurrencyNotSupported
# vs
raise_error EcbRates::CurrencyNotSupported
```